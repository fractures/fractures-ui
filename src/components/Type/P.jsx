import PropTypes from "prop-types"
import styled from "styled-components"

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const P = styled.p`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 1rem;
	line-height: 1.5rem;

	& > a {
		font-weight: 700;
		text-decoration: underline;
	}

	& > cite {
		background-color: var(--color-yellow-500);
	}
`

P.propTypes = propTypes

export default P
