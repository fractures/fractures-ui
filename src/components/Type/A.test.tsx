import P from './P'

describe('P', () => {
	const tree = renderer.create(<P>Lorem ipsum</P>).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
