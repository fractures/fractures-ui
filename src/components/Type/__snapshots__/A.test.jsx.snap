// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`P Matches snapshot 1`] = `
<p
  className="sc-bdVaJa gDBvja"
>
  Lorem ipsum
</p>
`;
